### PR TITLE
Implement ISG

### PR DESCRIPTION
**Pull request by PhlexPlexico**
I saw this feature being asked for by a few people. Since it shouldn't directly interfere with gameplay, unless specifically activated (pickup item while shielding and swinging). I've checked both on patched and non-patched versions of the game to see if this implementation works, and they both had the same expected behaviour (8 slashes with kokiri sword to kill Gibdos). It appears that this implementation should not impact any regular game play.

### CODE REVIEW

File: hooks/rst_isg,hks [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Since ARMv6 we can just call LDR to the address
```
Comment (leoetlino) [inline hooks/rst_isg,hks:1]: Why not just use a (relative) branch with B? Absolute branches are kind of overkill IMO.

Comment (PhlexPlexico) [reply hooks/rst_isg,hks]: Fair point. I'm not good at math and learned a lot of new things. I believe this next commit should work with a relative branch, thanks for the feedback!

**Comment by leoetlino** [comment]
Whoops, sorry for forgetting about this! Has the relative branch been tested? I don't really have enough time to test changes so I'll trust you and merge if this has been tested after you converted the branches into relative jumps

**Comment by PhlexPlexico** [comment]
Aside from the comma existing instead of a period, I did test it once more and it seemed to pass all my cases. A few are listed:
- Make sure it can't be triggered through regular sword swinging/unintentionally.
- Other forms don't cause UB or crashing.
- Damage is still the same on crouch stabs/slashes/etc.

I believe this should be good to go!

**Review by leoetlino — APPROVED** [review]

**Pull request merged**